src/kirby/components/calendar/calendar.component.html

```diff
@@ -0,0 +1,31 @@
+<div id="calendar">
+  <div class="header">
+    <button [disabled]="!canNavigateBack" (click)="changeMonth(-1)">
+      <kirby-icon name="arrow-back"></kirby-icon>
+    </button>
+
+    <div class="month-and-year"><span class="month">{{ activeMonthName }}</span><span class="year">{{ activeYear }}</span></div>
+  
+    <button [disabled]="!canNavigateForward" (click)="changeMonth(1)">
+      <kirby-icon name="arrow-forward"></kirby-icon>
+    </button>
+  </div>
+
+  <table class="calendar-table">
+    <thead class="calendar-table-header">
+      <th *ngFor="let weekDay of weekDays">{{ weekDay }}</th>
+    </thead>
+      
+    <tbody>
+      <tr *ngFor="let week of month">
+        <td *ngFor="let day of week">
+          <div (click)="onDateSelected(day)" class="{{day.cssClasses}}" [class.selected]="day.isSelected">
```

Comment:
Is [class.selected] necessary? Isn't it already included in 'day.cssClasses' ??

Reply:
Yes, but only intially when calendar is rendered or when month changes. When clicking on a day/date it needs to update that css class for the currently selected item + the new one